packages/component-library/package.json

```diff
@@ -43,7 +43,7 @@
     "react-animate-height": "^2.0.15",
     "react-focus-lock": "^1.19.1",
     "react-media": "^1.9.2",
-    "react-select": "^3.0.8",
```

Comment:
This upgrade may not have been necessary, but I didn't want us to have two slightly different versions of react-select.

Reply:
This is for `@kaizen/component-library` and the other one is for `@kaizen/draft-select`. Ideally they would always be on the same version a regular refresh. Thanks for updating this! `react-select` is likely not even be used in `@kaizen/component-library` any more, and is probably a relic from when we moved components into draft packages. I'll create a github issue to remove unused packages from `packages/component-library/package.json`